lib/radiator_web/graphql/public/resolvers/session.ex

```diff
@@ -1,4 +1,4 @@
-defmodule RadiatorWeb.GraphQL.Resolvers.Session do
+defmodule RadiatorWeb.GraphQL.Public.Resolvers.Session do
```

Comment:
Authentication requests go against the public endpoint because the admin schema requires a valid token/auth. But I realize that the `get_authenticated_session` method is duplicated, the one in the Admin context is never used.

Reply:
Ah, I get it now. Okidoki.